sdk/keyvault/test-resources.json

```diff
@@ -98,6 +98,18 @@
     "KEYVAULT_URI": {
       "type": "string",
       "value": "[reference(variables('keyVaultName')).vaultUri]"
+    },
+    "CLIENT_OBJECT_ID": {
+      "type": "string",
+      "value": "[parameters('testApplicationOid')]"
+    },
+    "BLOB_STORAGE_URI": {
+      "type": "string",
+      "value": "7.2-preview-blob-storage-uri"
```

Comment:
These need more work. This package won't run live tests for now. I'm leaving this as a reminder of what I have to do later.

Reply:
Log an issue?

Reply:
I believe we don't have a Managed HSM test environment for the pipeline runs yet, so I think this should be ok.

Reply:
We have a proper fix in the roadmap, so I won't log another issue.